scripts/autoretrieve-service/autoretrieve-register.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Register Autoretrieve on Estuary
+
+[Service]
+Type=oneshot
+EnvironmentFile=/etc/estuary/config.env
+ExecStart=/usr/local/bin/autoretrieve register-estuary ${ESTUARY_API} ${ESTUARY_ADMIN_TOKEN}
```

Comment:
since we're using this explicit path, maybe `Makefile` needs an `install` target that installs it there?